docs: Introduced new chapter for gateways (#587)
kyma-project/telemetry-manager: docs/user/resources/README.md

# Telemetry Resources

> The API of the Telemetry module is based on Kubernetes Custom Resource Definitions (CRD) which are extending the Kubernetes API with custom additions. In the following the specification of the Telemetry module API can be introspected:

Reviewer: ```suggestion
The API of the Telemetry module is based on Kubernetes Custom Resource Definitions (CRD), which are extend the Kubernetes API with custom additions. To inspect the specification of the Telemetry module API, see:
```
